templates/install_rvm.sh.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+USER=<%= @user %>
+
+SERVERS=(
+    hkp://keys.gnupg.net
+    hkp://pgp.mit.edu
+)
+
+for server in "${SERVERS[@]}"
+do
+    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
```

Comment:
What started this was the first server being unreachable. So if the key
ever changes we'd have to update whereas this should hopefully break never
again with the redundancy.
On Apr 7, 2015 4:43 PM, "Stephen Benjamin" <notifications@github.com> wrote:

> In templates/install_rvm.sh.erb
> <https://github.com/Katello/puppet-katello_devel/pull/36#discussion_r27918207>
> :
>
> > @@ -0,0 +1,14 @@
> > +#!/bin/bash
> > +
> > +USER=<%= @user %>
> > +
> > +SERVERS=(
> > +    hkp://keys.gnupg.net
> > +    hkp://pgp.mit.edu
> > +)
> > +
> > +for server in "${SERVERS[@]}"
> > +do
> > +    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
>
> You could also just include the GPG key itself if there's some kind of
> network problem that's preventing the downloading. Up to you.
>
> (RVM's gpg signing is still of dubious value anyway if the script we're
> getting from https://get.rvm.io isn't signed itself. Nothing's stopping
> the @TheNSABackdoorCommittee <https://github.com/TheNSABackdoorCommittee>
> from doing a MITM attack there.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/puppet-katello_devel/pull/36/files#r27918207>.
>